### ISSUE TITLE
simple-grid vs widths?

### ISSUE THREAD

**Issue by scanieso** [freshtilledsoil/base-style-framework#1]
@scottaohara What is the use case difference between [simple-grid](https://github.com/freshtilledsoil/base-style-framework/blob/master/stylesheets/layout/_simple-grid.scss) and [widths](https://github.com/freshtilledsoil/base-style-framework/blob/master/stylesheets/layout/_widths.scss)?

**Comment by scottaohara** [comment]
the simple grid file is separate from the widths due to our conversation yesterday that people may want to use their own grids.

As I've seen in my studying of other frameworks, the columns (widths here) are usually closely tied to the grid system. Where as with what I've written, they can be used independently of the grid system.

I've found this to be quite useful when I need to set widths on specific elements (buttons, media, inputs) and instead of modifying the existing classes for those elements, or creating a brand new class, I've been able to use the width classes over and over again.

EDIT:

I realized that your question may have actually been in reference to the fact that I hadn't deleted an older version of the grid, so there were two variations of it in those files.  I've removed the one that shouldn't exist in commit [https://github.com/freshtilledsoil/base-style-framework/commit/7a88d88e5c214637cd20206b8f34435a92a807e2]
